src/spin1_beam_model/cst_processing.py

```diff
@@ -132,12 +132,11 @@ def compute_spin1_harmonics(self, filter_cut_factor=1.0):
             pos1_E = (Et + 1j * Ep) / np.sqrt(2.0)
             neg1_E = (Et - 1j * Ep) / np.sqrt(2.0)
 
-            pos1_Elm = pyssht.forward(
-                pos1_E, self.L_data, Spin=1, Method="MWSS", Reality=False
-            )
-            neg1_Elm = pyssht.forward(
-                neg1_E, self.L_data, Spin=-1, Method="MWSS", Reality=False
-            )
+            pos1_Elm = np.empty([self.L_data * self.L_data], dtype=complex)
```

Comment:
Can we give (brief) insight into these magic numbers with a comment? If that'd be too long or involved, we can also just point to the documentation of `mw_forward_sov_conv_sym_ss` (assuming docs exist...)

Reply:
Docs most definitely do not exist (yet)! I don't really know what the numbers mean, I just took them from the code of ssht itself. Not really sure where to point, or what to say! The relevant line is https://github.com/UPennEoR/ssht/src/python/pyssht.pyx#L258

Reply:
Okay, no worries then. Thanks for tracking it down. We can just add it to our ongoing crusade to improve the documentation on the repo.